docs/docs/getting-started/concepts.md

```diff
@@ -285,17 +317,19 @@ A <PyObject section="config" module="dagster" object="RunConfig" /> is a set sch
     style CodeLocation fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
 
 
-    Asset -.-> Definitions
-    AssetCheck -.-> Definitions
-    IOManager -.-> Definitions
-    Job -.-> Definitions
-    Resource -.-> Definitions
-    Schedule -.-> Definitions
-    Sensor -.-> Definitions
+    Asset -.-> Component
+    AssetCheck -.-> Component
+    IOManager -.-> Component
+    Job -.-> Component
+    Resource -.-> Component
+    Schedule -.-> Component
+    Sensor -.-> Component
```

Comment:
One thing that I'm not clear on from these changes is that _components_ are an opt-in layer on top of traditional definitions. Part of me wonders if some of these diagrams should be _with_ components and _without_. What do you think?

Reply:
I think it makes sense and to leave it out of the diagram from definitions. But maybe add something about components in the table and the description.